commercetools.NET/Common/Client.cs

```diff
@@ -10,6 +10,7 @@
 
 using Newtonsoft.Json;
 using Newtonsoft.Json.Linq;
+using System.Threading;
```

Comment:
Is this import still needed after the Thread.sleep change? Else everything seems to be okay.

Reply:
Thanks @jenschude, removed the reference and committed